api/lib/notifications.js

```diff
@@ -166,6 +166,8 @@ function getIcon({ eventType, data }) {
     case eventTypes.PROPOSAL_CREATE:
     case eventTypes.PROPOSAL_UPDATE:
       return `/img/icons/activity/Submitted.svg`
+    case eventTypes.VALIDATOR_ADDED:
+      return `/img/networks/${notificationData.nextValidator.networkId}.png`
```

Comment:
what image is this? i think we meant just remove the `VALIDATOR_ADDED` notification for users - but we don't want to delete the notification because it will be used for network profiles and validator profiles

Reply:
yes, this is the second part: ` remove new validator notification for now **(and use network icon instead of lunie logo - for future use)**`

I tried to do the first part but failed in a first attempt